infinity-app/app.R

```diff
@@ -397,7 +494,8 @@ server <- function(input, output, session) {
         ~ Lat,
         popup = paste(filtered_data()$Species, filtered_data()$`Voucher Type`)
       ) %>%
-      addPolygons(data = place_polygon, color = "red")
+      addPolygons(data = place_polygon, color = "red") %>%
```

Comment:
We need some colour blind friendly colours here - could be done is a new feature branch for sure

Reply:
@tmesaglio thoughts on colors?